package.json

```
@@ -64,11 +64,13 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/zooniverse/pandora",
   "scripts": {
-    "start": "BABEL_ENV=development webpack-dashboard -p 3001 -- webpack-dev-server --config ./webpack.config.js",
+    "_start": "NODE_ENV=staging BABEL_ENV=development webpack-dashboard -p 3001 -- webpack-dev-server --config ./webpack.config.js",
+    "start": "export HEAD_COMMIT=$(git rev-parse --short HEAD); npm run _start;",
```

could we inject this at build time via jenkinsfile https://github.com/zooniverse/pandora/blob/f1def66cfe21202fbd2222c7a8b0f54478e7ee24/Jenkinsfile#L22

and use a default env var with this fallback to git cmd? 
```suggestion
    "start": "export HEAD_COMMIT=${HEAD_COMMIT:-$(git rev-parse --short HEAD)}; npm run _start;",
```

---

I think that would work, with build args in the Dockerfile?

I copied the approach here from the PFE package file.
https://github.com/zooniverse/Panoptes-Front-End/blob/173d7502fb0802e3cdf63b90ca5e4691822b87c9/package.json#L131

---

Force-pushed over that to alter the `build` script too. `start` is the local development server.